app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ProjectsController < ApplicationController
+  def id
```

Comment:
You don't need this action as a helper. Just use `params[:id]` in your `show` action.